spec/support/authentication_helpers.rb

```diff
@@ -27,6 +27,8 @@ def submit_credentials
     sleep 1
     click_button 'Login'
 
+    return if find('Pushed a login request to your device')
```

Comment:
Note that this will keep looking until our timeout for the text, which will slow things down if the text is NOT present ... can someone be more clever about this?